_im-plugin/ism/api-config-notice.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: default
+title: Configuring ISM notifications settings
+parent: ISM API
+grand_parent: Index State Management
+nav_order: 10
+---
+
+## Configuring ISM notifications settings
+
+Introduced 2.8
+{: .label .label-purple }
+
+Notifications settings are configured globally, so they do not need to be configured manually for each operation. Notifications settings allow users to control how they receive notifications about index events. For example, you can choose to receive email notifications, Slack channel notifications, or both. This documentation is intended for IT administrators as a guide to configuring notifications settings.
```

Comment:
@zhichao-aws Will the admins work from the Dev Tools console in OpenSearch Dashboards?

Reply:
Yes, work from the Dev Tools console is a practical choice.